plugins/src/ios/OptimoveSDKPlugin.swift

```diff
@@ -34,4 +34,94 @@ import NotificationCenter
         guard let config = OptimoveSDKPlugin.config else { return }
         Optimove.initialize(with: config)
     }
+    
+    @objc(reportEvent:)
+    func reportEvent(command: CDVInvokedUrlCommand) {
+        guard let name = command.arguments[0] as? String else {
+            let pluginResult = CDVPluginResult(status: .error, messageAs: "name is not invalid")
```

Comment:
Minor sp: "name is not valid" or "name is invalid" :)

Reply:
(see this copy/pasted in other places)

Reply:
fixed